apps/peppermint/docker-compose.arm64.yml

```diff
@@ -0,0 +1,58 @@
+version: "3.7"
+
+services:
+  peppermint:
+    image: pepperlabs/peppermint:dev
```

Comment:
In config.json it states latest so we should use this one. If arm is only available in `dev` we should rather drop arm support

Reply:
Its actually old so the dev would be preferred check the tags here https://hub.docker.com/r/pepperlabs/peppermint/tags.

Reply:
Dev indicates that it's a development build so not ready to be used. Also its not the same size of the other one so we can assume this app does not support arm

Reply:
Ok dropping support for it...